gammapy/cube/tests/test_psf_map.py

```diff
@@ -361,3 +361,25 @@ def test_make_mean_psf(data_store):
 
     assert not np.isnan(psf.psf_value.value).any()
     assert_allclose(psf.psf_value.value[22, 22], 12206.1665, rtol=1e-3)
+
+
+@pytest.mark.parametrize(
+    "position", [
+        SkyCoord("0 deg", "0 deg"),
```

Comment:
You can just put `["0d 0d", "180d 0d", ...]` here, and then put `SkyCoord(position)` once in the code below, might on one line then. (same in the test above).
And I think you have to add `@requires_data`.